flow/flow.go

```diff
@@ -1280,6 +1392,13 @@ func segmentProcess(parameters interface{}, inIndex []int32, stopper [2]chan int
 						}
 						safeEnqueue(OUT[index][inIndex[q]], OutputMbufs[index], uint(countOfPackets[index]))
 						currentState.V.Packets += uint64(countOfPackets[index])
+						if countersEnabled {
+							// FIXME: inaccurate statistics because
```

Comment:
Lets have TODO everywhere. Also I propose to have this comment in one place

Reply:
Removed